grunttasks/copyright.js

```diff
@@ -0,0 +1,26 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+module.exports = function (grunt) {
+  'use strict';
+
+  grunt.config('copyright', {
+    app: {
+      src: [
+        "{,bin/,config/,db/,scripts/}*.js"
+      ],
+      options: {
+        pattern: "This Source Code Form is subject to the terms of the Mozilla Public"
+      }
+    },
+    tests: {
+      src: [
+        'test/{remote,local,backend}/*.js'
```

Comment:
Unrelated: We can remove the "remote" since I don't see a [test/remote](https://github.com/mozilla/fxa-auth-db-mysql/tree/master/test) directory in this repo any more.